gems/pending/util/esx_thumb_print.rb

```diff
@@ -0,0 +1,34 @@
+require 'util/thumb_print'
+
+class ESXThumbPrint < ThumbPrint
+  attr_reader :user, :password, :cert
+
+  def initialize(host, user, password)
+    @user     = user
+    @password = password
+    super(host)
+  end
+
+  def uri
+    url  = "https://" + @host + "/host/ssl_cert"
+    @uri = URI(url)
+    @uri
```

Comment:
This last line is redundant.   The return value from `@uri = URI(url)` is the value anyway.